module-1/m14-wev-dev-environment/muder-mystery.md

```diff
@@ -4,4 +4,153 @@ author: Tuan Hoang
 description: solutions for full stack web assignments
 ---
 
-[Back to README](../README.md)
+
+# Murder Mystery
+Init: Open file README.md
+Open your terminal
+
+ 
+#### 00 
+1. `cd desktop/cmystery`
+2. `cat step0`
+
+3. `Cmd +N`: Open a new terminal to navigate around and testing new commandlines. 
+
+By the time, there are two opened Terminal, one for reading instructions one for testing commandlines and navigate files.
+
+![](https://i.imgur.com/jPQwZr5.png)
+
+
+Check up folder `Detective Handbook`
+Check up folder `mystery`
+
+
+#### 01
+
+------
+From: clmystery/
+------
+
+1. `cat step1`
+2. `less mystery/crimescene` : to view content in different viewer.
```

Comment:
less is not to view content on different viewer. I think you are trying to tell user to view content in different viewer, however, your wording here looks like an explanation for the `less m/c`

Reply:
Less is a command line utility that displays the contents of a file or a command output, one page at a time